vars/govuk.groovy

```diff
@@ -39,6 +39,17 @@ def buildProject(Map options = [:]) {
       defaultValue: false,
       description: 'Identifies whether this build is being triggered to test a change to the content schemas'
     ),
+    //not for use before content schemas have been merged into publishing api
+    booleanParam(
+      name: 'USE_PUBLISHING_API_FOR_SCHEMAS',
+      defaultValue: false,
+      description: 'DO NOT USE YET: Identifies whether to use publishing api or content schemas for schemas - flag to enable switchover'
```

Comment:
I feel a bit uneasy about merging this - i.e why merge something if it can't be used and then I wondered if it's actually needed if it's not supposed to be used and whether it could be an env var.

I came to an assumed conclusion that you probably need it as I'm expecting you'll need Publishing API to use this to trigger downstream builds during your testing and then probably will be removed. So I wonder if it should be relabelled from "DO NOT USE YET" to be like "TEMPORARY: Used to test migration of content schemas to Publishing API, to be removed after migration is completed"

I sort of wondered if you'd ever need the "yet" part of it? I guess it'd be available for testing and then we'd remove it rather than have a period where we encourage users to check it? But I could be misunderstanding.

Reply:
Hello, good point, I think I left this in from earlier when I hadn't fully settled on the order which I was going to merge things in. I'll rejig this description.